### DISCUSSION TITLE
installation DU CODE JJAZZJAB

### DISCUSSION THREAD

**Discussion by samedjon** [jjazzboss/JJazzLab#472] [General]
Bonjour chère communauté quelqu'un  Peux m'expliquer plus en détail comment procéder une fois que j'ai téléchargé le fichier zip et que j'ai décompresser, et que je l'ai ouvert avec NetBeans quelle est la prochaine étape pour pouvoir réussi à compiler le code et avoir une fenêtre qui s'ouvre et qui me présenter toutes tes fonctionnalités comment on fait pour m'assurer que tout le code que je vais télécharger entièrement fonctionnel est prêt à être modifié Peux-tu me piquer cela en détail s'il te plaît un battant du principe que j'utilise le dernier version de Netbeans Apache et que je ne suis pas un développeur, donc je n'ai pas de compétences poussés explique ma paca pas pour que j'y arrive s'il vous plaît

**Comment by jjazzboss** [comment]
Si vous n'êtes pas développeur ça va être compliqué... Il faut commencer par apprendre maven et netbeans. bon courage.
